adm_program/system/classes/Email.php

```diff
@@ -150,18 +158,22 @@ public function __construct()
      * in the email send process.
      * @param string $address A valid email address to which the email should be sent.
      * @param string $name    The name of the recipient that will be shown in the email header.
+     * @param array $additionalFields    Additional fields to map in a Key Value like Array. Not used at all yet.
      * @return bool Returns **true** if the address was added to the recipients list.
      */
-    public function addRecipient($address, $name = '')
+    public function addRecipient($address, $firstName = '', $name = '', $additionalFields = array())
```

Comment:
Have you test you changes? You change the parameters of the method but don't change the calls of the method. Also currently the "name" is "Firstname Lastname" and not only the lastname.